src/script/auth/module/reducer/clientReducer.ts

```diff
@@ -78,7 +78,7 @@ export function clientReducer(state: ClientState = initialClientState, action: A
     case CLIENT_ACTION.CLIENT_REMOVE_SUCCESS: {
       return {
         ...state,
-        clients: [...state.clients.filter(client => client.id === action.payload)],
+        clients: [...state.clients.filter(client => client.id !== action.payload)],
```

Comment:
I think what we want is to filter removed client out of the list, not to make the list include only removed client

Reply:
It's normally not visible for the user as we automatically proceed to login flow.